pydeequ/checks.py

```diff
@@ -100,6 +99,14 @@ def __init__(self, spark_session: SparkSession, level: CheckLevel, description:
         for constraint in self.constraints:
             self.addConstraint(constraint)
 
+    @staticmethod
+    def _handle_invalid_column(column):
+        """
+        :param column: column to which constraint is to be applied
+        """
+        if "." in column:
```

Comment:
I think this is quite strict, perhaps simply enclosing the field with ticks will do, like ``f"`column.name`"``

Reply:
@herminio-iovio  passing through ticks also does not work and returns the same error